### DISCUSSION TITLE
green screen with 4K version A380

### DISCUSSION THREAD

**Discussion by MV-Mike** [flybywiresim/aircraft#9251] [Help]
### Aircraft Version

Development

### Build info

```json
1
```

### Describe the bug

green screen, starting from DXB (inibuild) with 4k A380 version. trying now 8k version

### Expected behavior

wanted to fly from dxb, not possible

### Steps to reproduce

1. leg from dxb to rors 
2. dxb airport from inibuid
3. A380 4K Version
4. green screen, pc restarting
5. trying 8k version after deleting 4k version, everytime using the newest FbW Updater

**Comment by alepouna** [comment]
Hey, this seems to be a technical issue with the configuration of the aircraft rather than a bug. We recommend you try to reach out via our Discord Server first (https://discord.gg/flybywire) and reach out to our support channels! It will allow for easier communication and hopefully resolve the issue. 

*This issue will therefore be closed and be transferred to a discussion, for if you are unable to join our Discord community.*
